change/@fluentui-react-breadcrumb-preview-3e3c117d-535c-4eb4-8b75-7c46c919bc35.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29096&commitId=fee4a5ea90af28170b3858582dc968a65e4cdd5b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Breadcrumb Converged</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Breadcrumb Converged.appearance - Dark Mode.focused.chromium.png|680| Changed|
|Breadcrumb Converged.appearance - High Contrast.default.chromium.png|1099| Changed|
|Breadcrumb Converged.appearance.default.chromium.png|756| Changed|
|Breadcrumb Converged.appearance - RTL.focused.chromium.png|753| Changed|
|Breadcrumb Converged.appearance - Dark Mode.hover.chromium.png|680| Changed|
|Breadcrumb Converged.appearance - Dark Mode.default.chromium.png|680| Changed|
|Breadcrumb Converged.appearance - RTL.pressed.chromium.png|753| Changed|
|Breadcrumb Converged.appearance - High Contrast.pressed.chromium.png|1099| Changed|
|Breadcrumb Converged.appearance - Dark Mode.pressed.chromium.png|680| Changed|
|Breadcrumb Converged.appearance.pressed.chromium.png|756| Changed|
|Breadcrumb Converged.appearance - High Contrast.hover.chromium.png|1099| Changed|
|Breadcrumb Converged.appearance - RTL.default.chromium.png|753| Changed|
|Breadcrumb Converged.appearance - High Contrast.focused.chromium.png|1099| Changed|
|Breadcrumb Converged.appearance - RTL.hover.chromium.png|753| Changed|
|Breadcrumb Converged.divider type.default.chromium.png|814| Changed|
|Breadcrumb Converged.appearance.focused.chromium.png|756| Changed|
|Breadcrumb Converged.divider type.focused.chromium.png|814| Changed|
|Breadcrumb Converged.appearance.hover.chromium.png|756| Changed|
|Breadcrumb Converged.size.focused.chromium.png|1970| Changed|
|Breadcrumb Converged.size.hover.chromium.png|1970| Changed|
|Breadcrumb Converged.divider type.hover.chromium.png|814| Changed|
|Breadcrumb Converged.size.default.chromium.png|1970| Changed|
|Breadcrumb Converged.size.pressed.chromium.png|1970| Changed|
|Breadcrumb Converged.divider type.pressed.chromium.png|814| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>